exercises/shared/.docs/tests.md

```diff
@@ -0,0 +1,3 @@
+# Tests
+
+The exercises are written so that when you compile your program and run it it will execute the tests.  No special command line flags or other tomfoolery is required.
```

Comment:
Maybe you could add an example of what the actual command would be to run the tests?

Reply:
Done!